source/includes/steps-fle-convert-to-a-remote-master-key-azure.yaml

```diff
@@ -230,6 +255,31 @@ content: |
            To use the Azure Key Vault, you must use `pymongocrypt <https://pypi.org/project/pymongocrypt/>`__
            version 1.1 or later in your application's environment.
 
+     .. tab::
+        :tabid: csharp
+
+        .. code-block:: csharp
+
+           var clientEncryption = new ClientEncryption(clientEncryptionOptions);
+           var azureKeyName = Environment.GetEnvironmentVariable("FLE_AZURE_KEY_NAME");
+           var azureKeyVaultEndpoint = Environment.GetEnvironmentVariable("FLE_AZURE_KEYVAULT_ENDPOINT"); // typically <azureKeyName>.vault.azure.net
+           var dataKeyOptions = new DataKeyOptions(
+               masterKey: new BsonDocument
+               {
+                   { "keyName", azureKeyName },
+                   { "keyVaultEndpoint", azureKeyVaultEndpoint }
```

Comment:
`keyVersion` is missed. See here: https://github.com/mongodb/specifications/blob/master/source/client-side-encryption/client-side-encryption.rst#masterkey